IntelliJIdea2019/Airlift.xml

```diff
@@ -144,7 +144,6 @@
     <option name="THROWS_LIST_WRAP" value="1" />
     <option name="EXTENDS_KEYWORD_WRAP" value="2" />
     <option name="THROWS_KEYWORD_WRAP" value="2" />
-    <option name="KEEP_SIMPLE_BLOCKS_IN_ONE_LINE" value="true" />
```

Comment:
Should this be `false` instead of removed?

Reply:
You are right, it would state explicitly our preferences.